Containerfile.rag

```diff
@@ -0,0 +1,20 @@
+# vim: set filetype=dockerfile
+
+FROM quay.io/openshift/lightspeed-service-api:latest
+
+USER 0
+
+ARG RAG_INDEX=https://github.com/ilan-pinto/lightspeed-rag-documents/releases/latest/download/local.zip
+
+RUN microdnf install -y unzip wget
+
+RUN mkdir -p vector-db/ocp-product-docs \
+  && cd vector-db \
+  && wget ${RAG_INDEX} \
+	&& unzip local.zip \
+	&& mv local vector-db \
+	&& rm -f local.zip
+
+RUN microdnf remove -y unzip wget
```

Comment:
if you want this to be useful you need to do it in the same RUN command as the one where you installed it.  Otherwise you're just adding another layer to the image where these files are masked, but they're still in the lower layers.

Reply:
Ah, interesting. Will move.